doc/content/integrations/node-red/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Node-RED"
+description: ""
+---
+
+[Node-RED](https://nodered.org/) is a free, JavaScript-based development tool for visual programming, developed to ease the process of wiring together hardware devices, APIs and online services.
+
+The MQTT server that is exposed by {{% tts %}} can be connected to Node-RED. This integration allows setting up a Node-RED flow that listens to events and uplink messages, and it is also possible to schedule downlink messages.
+
+{{< figure src="demo_setup.png" alt="Demo setup" >}}
```

Comment:
Not sure we should put the image in the summary

Reply:
@johanstokking mentioned adding a cropped screenshot with a demo setup here. Should I remove this?

Reply:
I don't know if @johanstokking meant in the summary or on the overview page

Reply:
I meant in the overview page, but under the more-tag, so it's not in the summary but in the overview page